src/udf/src/lib.rs

```diff
@@ -38,11 +38,15 @@ impl ArrowFlightUdfClient {
 
     /// Check if the function is available and return the function ID.
     pub async fn check(&self, name: &str, args: &Schema, returns: &Schema) -> Result<FunctionId> {
-        let mut path = vec![name.to_string()];
-        for arg in &args.fields {
-            path.push(format!("{}", arg.data_type()));
+        // path = name/[args,]*
+        let mut path = name.to_string() + "/";
+        for (i, arg) in args.fields.iter().enumerate() {
+            if i != 0 {
+                path += ",";
+            }
+            path += &arg.data_type().to_string().to_lowercase();
         }
```

Comment:
What about extracting this snippet to make the naming consistent everywhere? Like...
https://github.com/risingwavelabs/risingwave/blob/896d92ee3cad6095e021f74b0686777f3d5a33c6/src/frontend/src/handler/create_function.rs#L86-L97

Reply:
Good idea! But this snippet will be deprecated soon. I'm going to decouple the function name between kernel and UDF server in the next PR. The syntax of CREATE FUNCTION will be changed correspondingly to:
```sql
create function gcd(int, int) returns int language python as gcd using link 'http://localhost:8815';
```
where `gcd(int,int)` is the name in kernel and `gcd` is the name in UDF server.

This approach refers to Flink and I think it's better than force unified name. 🤔